src/docs/sdk/event-payloads/contexts.mdx

```diff
@@ -693,6 +693,23 @@ Additional information that allows Sentry to connect multiple transactions, span
 
 : _Optional_. The origin of the trace indicates what created the trace. For more details, see <Link to="/sdk/performance/trace-origin">trace origin</Link>.
 
+`data`
+
+: _Optional_. The map of custom provided data. Currently Sentry SDKs can set following fields:
+
+| Field            | Type              | Description                                                  |
+| ---------------- | ----------------- | ------------------------------------------------------------ |
+| `route`          | `string` or `map` | The current route in the application.                        |
+| `previous_route` | `string` or `map` | The previous route in the application the user was visiting. |
+
+The `route` can also have currently following predefined fields if it set to the map:
+
+| Field    | Type     | Description                                    |
+| -------- | -------- | ---------------------------------------------- |
+| `name`   | `string` | The name of the route in the user application. |
+| `params` | `map`    | Params assigned to this route.                 |
+
+If the route set to string it will be normalization and the above created object will be created assiging provided route name to the `route.name` field.
```

Comment:
```suggestion
If the route is set to a string (e.g. `"route": "foo"`), it will be normalized into a map (e.g. `"route": {"name": "foo"}`) server-side.
```